src/js/components/covid19/recipient/AwardFilterButtons.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * AwardFilterButtons.jsx
+ * Created by Jonathan Hill 07/06/20
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import AwardFilterButton from './AwardFilterButton';
+
+const propTypes = {
+    onClick: PropTypes.func,
+    filters: PropTypes.array,
+    activeFilter: PropTypes.string
+};
+
+const AwardFilterButtons = ({ onClick, filters, activeFilter }) => (
+    <div className="award-filter__buttons">
+        {
+            filters.map((button) => (
+                <AwardFilterButton
```

Comment:
At least I would suggest defining AwardFilterButton here in this file. These new files seem like they could just be in the container TBH.

If we're going to create an entire new component or two components for this, lets do it with an eye toward reusability. ATM we already have like five variations of buttons each created for its unique use case. Really we should just have a single button definition with `className` prop, `active` prop, `onClick` prop etc...

Mapping over these buttons could just be done by the caller.